### DISCUSSION TITLE
Group 17 discussion

### DISCUSSION THREAD

**Discussion by hannahlwang** [Digital-Preservation-Coalition/PythonStudyGroups#28] [General]
Setting up a discussion thread for group 17

**Comment by ccmn98** [comment]
I wanted to provide some links to some python related resources that I've found useful:

- Tutorials
-[ RealPython](https://realpython.com/)
-[ GeeksforGeeks](https://www.geeksforgeeks.org/python-programming-language/)

-A python library database
--[ PyPI](https://www.geeksforgeeks.org/python-programming-language/)  for using pip to install. There are a lot of libraries out there, so when you find one that you like and want to use for a project check it's github page or the website for the library to see if something is sketchy (weird wording or phrasing, no one .  I heard of some library's actually being malware instead. I haven't run into any library's like this . . . but they do exist. 

I just came across this website that shows you how healthy a python library/package is: [https://snyk.io/advisor/packages/python/popular](https://snyk.io/advisor/packages/python/popular)
